convert.py

```diff
@@ -250,15 +236,14 @@ def loadHFTransformerJson(model: LazyModel, config_path: Path) -> Params:
         )
 
     # LLaMA v2 70B params.json
-    # {"dim": 8192, "multiple_of": 4096, "ffn_dim_multiplier": 1.3, "n_heads": 64, "n_kv_heads": 8, "n_layers": 80, "norm_eps": 1e-05, "vocab_size": -1
+    # {"dim": 8192, "ffn_dim_multiplier": 1.3, "n_heads": 64, "n_kv_heads": 8, "n_layers": 80, "norm_eps": 1e-05, "vocab_size": -1
```

Comment:
Actually, why are we removing this?

Reply:
because it became irrelevant, but I guess removing it form the example is a bad idea.